src/types.ts

```diff
@@ -2,6 +2,16 @@ export type LuaState = number
 
 export type EnvironmentVariables = Record<string, string | undefined>
 
+export interface CreateEngineOptions {
+    openStandardLibs?: boolean
+    injectObjects?: boolean
+    enableProxy?: boolean
```

Comment:
JS objects in lua are not translated to tables, instead they are proxied through userdata, useful to classes, etc